addons/snailmail_account/wizard/account_invoice_send_views.xml

```diff
@@ -9,14 +9,17 @@
                 <xpath expr="//div[@name='option_email']" position='before'>
                     <div name="option_letter">
                         <field name="letter_ids" invisible="1"/>
-                        <field name="currency_id" invisible="1"/>
                         <div name="option" class="text-left d-inline-block">
                             <field name="snailmail_is_letter" />
                             <b><label for="snailmail_is_letter"/></b>
                         </div>
                         <div class="text-left d-inline-block mr8" name='cost' attrs="{'invisible': [('snailmail_is_letter','=', False)]}">
                             <b>(
-                                <field name="snailmail_cost" class="mr4" widget='monetary' options="{'currency_field': 'currency_id'}"/>
+                                <span>
+                                    <field name="snailmail_cost" options="{'digits':[0,0]}" class="mr4"/>
+                                    <span attrs="{'invisible': [('snailmail_cost', '&gt;=', 2)]}">Stamp </span>
```

Comment:
wont work well on other languages.
See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

Note that odoo does not currently support plurial form in translations

Reply:
@KangOl SOrry for the field thing, I was backporting something we planned tor master and forgot about it. What would you suggest as a solution for the plural form ?

Reply:
Only keeping plurial form should be ok.